### PR TITLE
Enterprise Theme Customizer

### PR DESCRIPTION
**Pull request by dulanism**
Building a new page the covers the capabilities of color theme customization and custom notifications

### CODE REVIEW

File: pages/enterprise/themes.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+###############################
+Custom Themes and Notifications
+###############################
+
+************
+Introduction
+************
+
+The Theme Customizer for Graylog Enterprise gived you visual control of your instance. Consider ways it's used
+to meet your needs. Administrators can:
```
Comment (waab76) [inline pages/enterprise/themes.rst]: Not really sure what the "Consider ways it's used to meet your needs." sentence is really doing there.  Also, typo on "gived" should be "gives".

```suggestion
The Theme Customizer for Graylog Enterprise gives you control over the color scheme of your instance. Administrators can:
```

File: pages/enterprise/themes.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+*  change the look and feel to meet company brand guidelines
+*  apply customer headers and notifications to display messages
```
Comment (waab76) [inline pages/enterprise/themes.rst]:
```suggestion
*  add custom headers and notifications to display messages 
```

File: pages/enterprise/themes.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+.. image:: /images/theme_hex_box.png
+
+********************
+Custom Notifications
+********************
+
+Imagine a scenario where you, a Graylog administrator, are tasked to notify your peers of a scheduled outtage.
+One way to get the message across is to use custom notifications. This is also available in the *Customization*
+menu.
```
Comment (waab76) [inline pages/enterprise/themes.rst]:
```suggestion
Custom notifications allow Graylog administrators to share messages with all of their Graylog users.  Whether you need to notify users of planned downtime or provide a warning message on the logon screen for compliance reasons, custom notifications will help you get the message out.  This feature is also available in the *Customization* menu.
```

**Review by waab76 — CHANGES_REQUESTED** [review]
The typo and confusing extra sentence in the introduction are the biggest issues for me.  Made a suggestion about rephrasing things for custom notifications, but it's not something I'd hold up the review for.

File: pages/enterprise/themes.rst (hunk 1 above)
Comment (dulanism) [reply pages/enterprise/themes.rst]: Yeah, that definitely leans toward blog-writing. I'll just remove the sentence, and fix that typo.

File: pages/enterprise/themes.rst (hunk 3 above)
Comment (dulanism) [reply pages/enterprise/themes.rst]: Updated

File: pages/enterprise/themes.rst (hunk 2 above)
Comment (dulanism) [reply pages/enterprise/themes.rst]: updated

File: pages/enterprise/themes.rst (hunk 1 above)
Comment (dulanism) [reply pages/enterprise/themes.rst]: I like it. Updated.

**Review by waab76 — APPROVED** [review]
LGTM!

**Pull request merged**